breakpad-symbols/src/sym_file/mod.rs

```diff
@@ -62,9 +60,9 @@ impl SymbolFile {
         let mut fully_consumed = false;
         loop {
             // Read the data in, and tell the circular buffer about the new data
-            let size = input_reader.read(buf.space()).map_err(|e| {
-                SymbolError::LoadError(format_err!("couldn't read input stream {}", e))
-            })?;
+            let size = input_reader
+                .read(buf.space())
+                .map_err(SymbolError::LoadError)?;
```

Comment:
Doesn't the `#[from]` annotation you included in the declaration of this variant generate a `From<io::Error>` impl, and thus make this work without `map_err`?